index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body class="container">
+    <div class="header_container">
+        <div>
+            <input type="image" src="MyLogo.svg" alt="MyLogo">
+        </div>
+        <img src="menu.svg" alt="Menu">
+    </div>
+    <img class="arc" src="arc.svg" width="1" height="15" alt="arc">
+    <img class="bar" src="bar.svg" width="1" height="15" alt="bar">
+    <img class="c" src="c.svg" width="1" height="15" alt="curve">
+    <img class="circle" src="circle.svg" width="1" height="15" alt="circle">
+    <div>
+        <h2 class="info_box">
+            I'm Zedd Glad to see you!
+        </h2>
```

Comment:
- [ ] [REQUIRED] Please, make sure to use `<h1>` instead of `<h2>` for semantic purposes

Reply:
Good job in implementing the changes 👍